### PR TITLE
Bring in peep.sh from fjord to deal with pip version incompatibilities.

### PR DESCRIPTION
**Pull request by mythmon**
@willkg r?

### CODE REVIEW

File: peep.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# argument.
+#
+# Note: If we encounter other versions of pip here that require other
+# things, we can toss them in.
+case $PIPVER in
+    1.5*|1.6*|6.0*)
+        echo "peep.sh: Wheel-using pip detected, so passing --no-use-wheel."
+        ARGS="--no-use-wheel"
+        ;;
+    6.1*)
```
Comment (willkg) [inline peep.sh]: I think given you know 6.0 works and it's covered in the previous case and you know 6.1 does not work, you can probably assume all future versions of pip won't work (for now), so you could just do:

```
6*)
```

Comment (willkg) [inline peep.sh]: Plus:

```
6*)
```

kind of looks like a :smiley:.

**Comment by willkg** [comment]
This looks good to me. r+

File: peep.sh (hunk 1 above)
Comment (mythmon) [inline peep.sh]: I suspect that future versions of pip might start working, actually, due to pypa/pip#2655. Either way, I plan on watching to see what happens with pip and peep.

**Pull request merged**